README.md

```diff
@@ -252,3 +252,5 @@ int main() {
 ## Project status
 
 Pistache hasn't yet hit the 1.0 release. This means that the project is _unstable_ but not _unusable_. In fact, most of the code is production ready; you can use Pistache to develop a RESTful API without issues, but the HTTP client has a few issues in it that make it buggy.
+
+<b id="linux-only">\*</b> While most code uses modern C++, Pistache makes use of some Linux-specific APIs where the standard library doesn't provide alternatives, and works only on that OS. See [#6](https://github.com/pistacheio/pistache/issues/6#issuecomment-242398225) for details. If you know how to help, feel free to open an issue!
```

Comment:
There are already lots of open issues.  I would reword to "please contribute a PR to add support for your desired platform."  Note that such a platform will need automated unit test coverage, which might complicate things.

Reply:
Yeah that's better, I didn't know how to word it, thanks :)

Regarding unit tests I think that's a smaller issue. The tests would be the same, as the system calls are an implementation detail; it would be sufficient to run the tests on the supported OS (correct me if I'm wrong)